client/app/account/user/advanced/user-advanced.html

```diff
@@ -1,85 +1,75 @@
-                               href="https://github.com/ovh-ux"
-                               class="align-bottom">
-                                https://github.com/ovh-ux
-                            </a>
-                        </li>
-                        <li class="mb-2">
-                            <span class="fa fa-2x fa-comments text-center mr-2"
-                                  aria-hidden="true">
-                            </span>
-                            <a target="_blank"
-                               href="https://gitter.im/ovh/ux"
-                               class="align-bottom">
-                                https://gitter.im/ovh/ux
-                            </a>
-                        </li>
-                    </ul>
-                </form>
+        <form>
+            <div class="row my-5">
+                <div class="col-xs-12 col-md-4 text-center">
+                    <span class="fa fa-desktop fa-3x d-inline-block mb-3"
+                            aria-hidden="true">
+                    </span>
+                    <p data-translate="user_account_advanced_section_developer_interface_opensource"></p>
+                </div>
+                <div class="col-xs-12 col-md-4 text-center">
+                    <span class="fa fa-puzzle-piece fa-3x d-inline-block mb-3"
+                            aria-hidden="true">
+                    </span>
+                    <p data-translate="user_account_advanced_section_developer_component_library"></p>
+                </div>
+                <div class="col-xs-12 col-md-4 text-center">
+                    <span class="fa fa-code fa-3x d-inline-block mb-3"
+                            aria-hidden="true">
+                    </span>
+                    <p data-translate="user_account_advanced_section_developer_api"></p>
+                </div>
+            </div>
 
+            <div class="oui-box oui-box_light">
+                <div class="oui-switch oui-switch_inline">
+                    <label class="oui-switch__label-container">
+                        <input type="checkbox"
+                                class="oui-switch__checkbox"
+                                data-ng-model="advancedCtrl.developmentMode.enabled"
+                                data-ng-change="advancedCtrl.updateDevelopmentMode()"
+                                data-ng-disabled="advancedCtrl.isLoadingDeveloperMode"
+                                data-track-on="click"
+                                data-track-type="action"
+                                data-track-name="account::advanced_settings::edit::developer_mode">
+                        <span class="oui-switch__slider" aria-hidden="true"></span>
+                    </label>
+                </div>
+                <span data-translate="user_account_advanced_section_developer_label"></span>
             </div>
-        </div>
 
+            <h3 class="mt-5"
```

Comment:
Applying margins always in the same direction would be useful to harmonize them across the UI.

I think we used to stick with `margin-bottom` as much as possible. So maybe you can apply this
margin to the parent element, right?